src/Resources/test/.php_cs.test

```diff
@@ -8,6 +8,7 @@ return (new PhpCsFixer\Config())
         '@Symfony' => true,
         '@Symfony:risky' => true,
         'native_function_invocation' => false,
+        'blank_line_before_statement' => ['statements' => ['break', 'case', 'continue', 'declare', 'default', 'do', 'exit', 'for', 'foreach', 'goto', 'if', 'include', 'include_once', 'phpdoc', 'require', 'require_once', 'return', 'switch', 'throw', 'try', 'while', 'yield', 'yield_from']],
```

Comment:
we need `phpdoc` to be added... TODO: check whats included in symfony rules set and only add the one we need.

Reply:
im using are test config here because, well lazy is cool, but thinking we either duplicate this for generating templates or rename / move this away from "test" and still use it for both ci and generating templates